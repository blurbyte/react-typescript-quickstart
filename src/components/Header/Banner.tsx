import styled from '@styles/styledComponents';
import { Colors } from '@styles/colors';

const Banner = styled.header`
  position: fixed;
  background: ${Colors.white};
  width: 100%;
  z-index: 99;
  top: 0;
  left: 0;
  border-bottom: 1px solid ${Colors.gray};
  box-shadow: 0 4px 4px -1px rgba(40, 40, 40, 0.1);
`;

export default Banner;
