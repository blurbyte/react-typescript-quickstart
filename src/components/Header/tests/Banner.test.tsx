import * as React from 'react';
import { shallow } from 'enzyme';

import Banner from '../Banner';

describe('Banner', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Banner />);
    expect(wrapper).toMatchSnapshot();
  });
});
