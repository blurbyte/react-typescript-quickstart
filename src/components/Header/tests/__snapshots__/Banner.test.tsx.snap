// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
.c0 {
  border-bottom: 1px solid #908e89;
  box-shadow: 0 4px 4px -1px rgba(40,40,40,0.1);
}

<header
  className="c0"
/>
`;
